EIPS/eip-5023.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 5023
+title: Shareable non-transferable non-fungible token
```

Comment:
For now there is a 44 letter length limit.  

@yaruno if you are still pursuing this EIP, can you leverage the EIP Walidator's warning for to make it conform to the latest EIP editorial requirement?

Reply:
Yes, we are still pursuing this EIP. I'll check the EIP validator warnings and update the PR accordingly.